### PR TITLE
Update Location And Role Error Message

### PR DESCRIPTION
**Pull request by HM33-Stu**
Update Location And Role Error Message
Related PR (update to UG): https://github.com/AY2324S2-CS2103T-W11-1/tp/pull/254

### CODE REVIEW

**Comment by HM33-Stu** [comment]
Updated Error Messages;
image
image

**Comment by LWS49** [comment]
UG fix for location not present, refer to issue

File: src/main/java/seedu/address/model/internship/Location.java [hunk 1]
```diff
@@ -7,7 +7,8 @@
  * Represents an Internship's location in the internship data.
  */
 public class Location {
-    public static final String MESSAGE_CONSTRAINTS = "Locations have to be either local, overseas, or remote";
+    public static final String MESSAGE_CONSTRAINTS = "Locations have to be either unknown, local, overseas, or remote. "
+            + "Else, it will be set to the default value of UNKNOWN";
```
Comment (jieqiboh) [inline src/main/java/seedu/address/model/internship/Location.java:11]: Is this considered making the message more specific?

**Review by CaptainKeqing — APPROVED** [review]
Looks good to me. Bug fixes for incorrect error msgs

**Pull request merged**